multigeiger/userdefines-example.h

```diff
@@ -41,6 +41,12 @@
 // Play a start sound at boot/reboot time?
 #define PLAY_SOUND true
 
+// Play an alarm sound when radiation level too high?
```

Comment:
... is too high?

Reply:
Added some further text and moved the alarm block to the end, as in IotWebConf